perllib/FixMyStreet/App/Form/Waste/Bulky/Shared.pm

```diff
@@ -70,6 +70,15 @@ has_page summary => (
             tandc => { option_label => $label }
         };
     },
+    field_ignore_list => sub {
+        my $page = shift;
+        my $c = $page->form->c;
+
+        if (!($c->cobrand->moniker eq 'sutton' || $c->cobrand->moniker eq 'kingston') || !$c->stash->{is_staff}) {
+            $page->form->{data}->{payment_method} = 'Credit card';
```

Comment:
What does this line do? `payment_method` would be set to `credit_card` I would assume, anyway, so I'm hoping it's not needed.